scripts/publish.sh

```diff
@@ -8,6 +8,7 @@ name="ed-extension-layer"
 ext_path="bin/extensions/$name"
 zip_name="extension.zip"
 
+rm -rf bin
```

Comment:
this is a dangerous command, better to use `cd` or stacked path in the script to create an isolated folder because you can call this script from anywhere.

Reply:
fixed